docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -81,6 +81,17 @@ Create monitors in {kib} by using your local journeys.
 npx @elastic/synthetics push --auth <api-key> --url <kibana-url> --project <id|name>
 ----
 
+[NOTE]
+====
+The `push` command includes interactive prompts to prevent you from accidentally deleting or duplicating monitors.
+You will see a prompt when:
+
+* You `push` a project that used to contain one or more monitors but
```

Comment:
Might need to reword to make it clear that `Monitors that are part of a particular project-id gets deleted` not all of them.